### PR TITLE
Fe story building superadmin

### PR DESCRIPTION
**Pull request by imranhayat**

### CODE REVIEW

**Comment by sdnorm** [comment]
@Moiz1524 

Functionality looks good. 

There is a confusing part but it might be cleared up when the FE PR is merged. The Builders show page seems lacking in that it only shows the title to the questions.

File: app/javascript/controllers/preview_controller.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { Controller } from "@hotwired/stimulus"
+
+export default class extends Controller {
+  static targets = ["inputPreText", "outputPreText", "inputPostText", "outputPostText"];
+
+  connect() {
+    this.updatePreTextOutput();
```
Comment (Moiz1524) [inline app/javascript/controllers/preview_controller.js:7]: This is also running on Builders#new form. We don't need to call the handlers when the prompt form is initially blank. I think it's better to exclude these calls for Builders#new page.

**Review by Moiz1524 — CHANGES_REQUESTED** [review]
Provided inline reviews

File: app/javascript/controllers/preview_controller.js (hunk 1 above)
Comment (imranhayat) [reply app/javascript/controllers/preview_controller.js]: Condition added if value is not present in input field, these functions will not be executed.

**Review by Moiz1524 — APPROVED** [review]

**Pull request merged**